pkg/interfaces/contracts/vault/IAggregatorCompositeLiquidityRouter.sol

```diff
@@ -0,0 +1,141 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+pragma solidity ^0.8.24;
+
+import { ICompositeLiquidityRouterQueries } from "./ICompositeLiquidityRouterQueries.sol";
+
+/**
+ * @notice An aggregator composite liquidity router for liquidity operations on ERC4626 and nested pools.
+ * @dev This contract allows interacting with ERC4626 Pools (which contain wrapped ERC4626 tokens) using only standard
+ * underlying tokens. For instance, with `addLiquidityUnbalancedToERC4626Pool` it is possible to add liquidity to an
+ * ERC4626 Pool with [waDAI, waUSDC], using only DAI, only USDC, or an arbitrary amount of both. If the ERC4626 buffers
+ * in the Vault have liquidity, these will be used to avoid wrapping/unwrapping through the wrapped token interface,
+ * saving gas.
+ *
+ * For instance, adding only DAI to the pool above (and assuming a waDAI buffer with enough liquidity), would pull in
```

Comment:
Not in the diff; you fixed it here, but `ICompositeLiquidityRouter.sol` still says "an aDAI buffer" here. That should be updated as well.